docs/reference/02-find.md

```diff
@@ -0,0 +1,82 @@
+# Basic API
+
+Find, update and delete records.
+
+The examples use the following prisma schema:
+
+```prisma
+model Post {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    updatedAt DateTime @updatedAt
+    published Boolean
+    title     String
+    content   String?
+
+    comments Comment[]
+}
+
+model Comment {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    content   String
+
+    post   Post @relation(fields: [postID], references: [id])
+    postID String
+}
+```
+
+## Reading data
+
+### Find many records
+
+```go
+posts, err := client.Post.FindMany(
+    db.Post.Title.Equals("hi"),
+).Exec(ctx)
+```
+
+If no records are found, this returns an empty array without returning an error (like usual SQL queries).
+
+### Find one record
+
+```go
+post, err := client.Post.FindOne(
+    db.Post.ID.Equals("123"),
+).Exec(ctx)
+
+if err == db.ErrNotFound {
+    log.Printf("no record with id 123")
+}
+```
+
+This returns an error of type `ErrNotFound` (exported in the `db` package) if there was no such record.
+
+### Query API
+
+Depending on the data types of your fields, you will automatically be able to query for respective operations. For example, for integer or float fields you might want to query for a field which is less than or greater than some number.
```

Comment:
A bit wordy, but I like the example. Maybe..

The query operations change based on the data types in your schema. For example, integers and floats will have greater than and less than operations, while strings have prefix and suffix operations.